.cirrus.yml

```diff
@@ -1,3 +1,19 @@
+environment:
```

Comment:
I have a concern about this change. When rolling out new builder images, I often set them to only run on PR then after a touch, I add to nightly and then finally use for release. 

This is going to be more difficult to juggle with this change.

I can see the reasoning behind it, but I had them separately indicated before because of this particular workflow. This change doesn't prevent it but does make it a touch more work to do.

Reply:
Happy to revert if you'd prefer.

Reply:
done